src/ipc_internal.h

```diff
@@ -21,8 +21,10 @@ extern "C" {
 // providerIpcData is a Flexible Array Member because its size varies
 // depending on the provider.
 typedef struct umf_ipc_data_t {
-    int pid;         // process ID of the process that allocated the memory
-    size_t baseSize; // size of base (coarse-grain) allocation
+    uint64_t handle_id; // unique handle ID
```

Comment:
Is it the unique ID of this handle? If yes, please replace with: `unique ID of this handle`

Reply:
> Is it the unique ID of this handle?

yes

> If yes, please replace with: unique ID of this handle

Done